{{cookiecutter.project_slug}}/compose/production/django/Dockerfile

```diff
@@ -125,9 +125,6 @@ RUN chown -R django:django ${APP_HOME}
 USER django
 
 RUN DATABASE_URL="" \
-  {%- if cookiecutter.use_celery == "y" %}
-  CELERY_BROKER_URL="" \
-  {%- endif %}
```

Comment:
I think that was required because the env variable has no default in the settings, which is still the case with `REDIS_URL`

Reply:
What do you think of this default: 
```python
env("REDIS_URL", default="redis://{% if cookiecutter.use_docker == 'n' %}localhost{%else%}redis{% endif %}:6379/0")
```

Reply:
Yes sounds good. Might read better with a "truthy" condition `if use_docker == y`? But yea, would be simpler with a default value in settings